src/util-cpu.c

```diff
@@ -77,7 +78,16 @@ uint16_t UtilCpuGetNumProcessorsConfigured(void)
 #elif OS_WIN32
 	long nprocs = -1;
 	const char* envvar = getenv("NUMBER_OF_PROCESSORS");
-	nprocs = (NULL != envvar) ? atoi(envvar) : 0;
+    if (envvar != NULL) {
+        if (ByteExtractStringInt64(&nprocs, 10, 0, (const char *)envvar) < 0) {
+            SCLogError(SC_ERR_INVALID_VALUE, "Invalid value for the number of "
+                       "cpus: '%s'", envvar);
+            return 0;
+        }
+    }
+    else {
```

Comment:
Suggest initializing `nprocs = 0`.

Remove the `else` corresponding to `if (envvar != NULL)`

Reply:
Is this what you're suggesting?
 ```
   81     if (envvar != NULL) {
   82         if (ByteExtractStringInt64(&nprocs, 10, 0, (const char *)envvar) < 0) {
   83             SCLogError(SC_ERR_INVALID_VALUE, "Invalid value for the number of "
   84                        "cpus: '%s'", envvar);
   85             return 0;
   86         }
   87     }
   88     nprocs = 0;
```